libs/wingsdk/src/target-tf-aws/counter.inflight.ts

```diff
@@ -40,6 +40,28 @@ export class CounterClient implements ICounterClient {
     return parseInt(newValue) - amount;
   }
 
+  public async dec(amount = 1): Promise<number> {
+    const command = new UpdateItemCommand({
+      TableName: this.tableName,
+      Key: { [HASH_KEY]: { S: COUNTER_ID } },
+      UpdateExpression: `SET ${VALUE_ATTRIBUTE} = if_not_exists(${VALUE_ATTRIBUTE}, :${INITIAL_VALUE_TOKEN}) - :${AMOUNT_TOKEN}`,
+      ExpressionAttributeValues: {
+        [`:${AMOUNT_TOKEN}`]: { N: `${amount}` },
+        [`:${INITIAL_VALUE_TOKEN}`]: { N: `${this.initial}` },
+      },
+      ReturnValues: "UPDATED_NEW",
```

Comment:
Why not return the old value here?

Reply:
If it's the first time returning old values ​​it doesn't work because there's nothing in the table